zh/backup-to-aws-s3-using-br.md

```diff
@@ -293,7 +293,7 @@ demo1-full-backup-s3   full   snapshot   Complete   s3://my-bucket/my-full-backu
 
     ```
     NAME                       TYPE    MODE   STATUS   ....
-    demo1-log-backup-s3        log    Running  ....
+    demo1-log-backup-s3                log    Running  ....
```

Comment:
The log backup CRs below all have no `TYPE` column, we should keep consistent.

Reply:
ok, I missed this.